docs/api-testing/environments.md

```diff
@@ -21,22 +21,22 @@ The **Environments** section lets you change anything, not just environments. Yo
 
 ## Creating Environments
 
-Once a domain is parametrized, you can override a variable if needed.
+Once you have a variable, you can override it if needed.
 
 There are multiple ways to create and access Test environments:
 
-### From a Project
+### From the Environments
```

Comment:
@lysannep we have a section called "Environments" inside each project (the description here), then users can create environments from different places inside the project (Test list, Composer, HTTP Client) all of them are available inside each project (next description below) so basically, all of those places are inside a project the only difference is in the "Environment" section you have a dedicated area to manage the environments whilst in the other places there's a drop-down for doing this...how we can differentiate those areas?

Reply:
@LoryApiFortress Under ## Creating Environments, we could add something like:

> You can create a test environment from:
> * the Environments page for a test project. 
> * the Run Configuration for a test project (from the Tests page) or a specific test (either from the Compose page when you create a test from scratch or the HTTP Client page when you create a test from an API call).

With this approach we could rename the H3 headings "From the Environments Page" and "From the Run Configuration".

Reply:
@lysannep 
> You can create a test environment from:
> 
> - the Environments page for a test project.
> - the Environment dropdown from the Tests page) or from the Compose page or the HTTP Client page 

Might work in this way? 

I changed your suggestion because:
- Run Configuration means nothing + Environment dropdown is also available in the load test (that is currently hidden for almost all the customers so I don't mention it in the list for the time being). In the load test, there's only the Environment dropdown, we don't have "Run Configuration" so we would need to change it again once it will be available for all customers.
- "either from the Compose page when you create a test from scratch" it's wrong because Environment dropdown is always available not only if you create a test from scratch
- "the HTTP Client page when you create a test from an API call" is wrong because you can select/create an Environment from the HTTP client and just call an endpoint without generating a test

Reply:
@LoryApiFortress Thanks for clarifying. Now that I have a better understanding, I think the headings are fine.